assets/src/app/ato/Documents/Fedramp.tsx

```diff
@@ -11,38 +11,32 @@ import * as Api from '@app/lib/api'
 interface FedRAMPsState {
     rows: IRow[];
 }
-class ComponentFedRAMPTemplates extends React.Component<any, FedRAMPsState> {
-    columns = ['Product', 'FedRAMP Low', 'FedRAMP Moderate', 'FedRAMP High'];
+
+interface FedRAMPsProps {
+    productID: string;
+}
+
+class ComponentFedRAMPTemplates extends React.Component<FedRAMPsProps, FedRAMPsState> {
+    columns = ['FedRAMP Low', 'FedRAMP Moderate', 'FedRAMP High'];
     constructor(props) {
         super(props);
-        Api.components()
-           .then(data => this.buildRows(data));
+        console.log("What is my productID::" + this.props.productID)
         this.state = {
-            rows: [],
+            rows: this.buildRows(this.props.productID),
```

Comment:
This builds the rows in the constructor once. When productId changes, the constructor is not re-run, only the render method is run.

For more info see: https://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/ and https://reactjs.org/docs/react-component.html#the-component-lifecycle 

I propose to remove state from this component.